web-server/v0.4/src/services/explore.js

```diff
@@ -19,3 +19,29 @@ export default async function querySharedSessions(params) {
     },
   });
 }
+
+export async function updateDescription(params) {
```

Comment:
The `updateDescription` function here take the edited value of description from the `user` and updates it in our database. Later after updating the database, we are using `querySharedSessions` method which is referred as`fetchSharedSessions` method in `pages/Explore/index.js` to re-display the newly updated data.

Reply:
Can you add the above as a comment in the code itself?